Detectors/TPC/qc/CMakeLists.txt

```diff
@@ -1,24 +1,55 @@
-# Copyright CERN and copyright holders of ALICE O2. This software is distributed
-# under the terms of the GNU General Public License v3 (GPL Version 3), copied
-# verbatim in the file "COPYING".
+#Copyright CERN and copyright holders of ALICE O2.This software is distributed
+#under the terms of the GNU General Public License v3(GPL Version 3), copied
+#verbatim in the file "COPYING".
 #
-# See http://alice-o2.web.cern.ch/license for full licensing information.
+#See http://alice-o2.web.cern.ch/license for full licensing information.
 #
-# In applying this license CERN does not waive the privileges and immunities
-# granted to it by virtue of its status as an Intergovernmental Organization or
-# submit itself to any jurisdiction.
+#In applying this license CERN does not waive the privileges and immunities
+#granted to it by virtue of its status as an Intergovernmental Organization or
+#submit itself to any jurisdiction.
 
 o2_add_library(TPCQC
                SOURCES src/PID.cxx
+                       src/Helpers.cxx
+                       src/TrackCuts.cxx
+                       src/Clusters.cxx
                PUBLIC_LINK_LIBRARIES O2::TPCBase)
```

Comment:
can you please add O2::DataFormatsTPC to the PUBLIC_LINK_LIBRARIES this should fix the compilation error